src/server/game/Events/GameEventMgr.cpp

```diff
@@ -1597,7 +1601,13 @@ void GameEventMgr::UpdateWorldStates(uint16 event_id, bool Activate)
     }
 }
 
-GameEventMgr::GameEventMgr() : isSystemInit(false) { }
+GameEventMgr::GameEventMgr() : isSystemInit(false)
+{
+}
+
+GameEventMgr::~GameEventMgr()
+{
+}
 
```

Comment:
why did you move the destructor from header over here when it's empty?

Reply:
This PR is largely a cherry-pick of my older PR to TC master. Shauren added to the PR before merge, and that was one of the changes for some reason. Codestyle I guess